keycloak/realm.go

```diff
@@ -48,6 +48,21 @@ type Realm struct {
 	InternationalizationEnabled bool     `json:"internationalizationEnabled"`
 	SupportLocales              []string `json:"supportedLocales"`
 	DefaultLocale               string   `json:"defaultLocale"`
+
+	//extra attributes of a realm, contains security defenses browser headers and brute force detection parameters(those still nee to be added)
+	Attributes Attributes `json:"attributes,omitempty"`
+}
+
+type Attributes struct {
+	BrowserHeaderContentSecurityPolicy           string `json:"_browser_header.contentSecurityPolicy,omitempty"`
+	BrowserHeaderContentSecurityPolicyReportOnly string `json:"_browser_header.contentSecurityPolicyReportOnly,omitempty"`
+	BrowserHeaderStrictTransportSecurity         string `json:"_browser_header.strictTransportSecurity,omitempty"`
+	BrowserHeaderXContentTypeOptions             string `json:"_browser_header.xContentTypeOptions,omitempty"`
+	BrowserHeaderXFrameOptions                   string `json:"_browser_header.xFrameOptions,omitempty"`
+	BrowserHeaderXRobotsTag                      string `json:"_browser_header.xRobotsTag,omitempty"`
+	BrowserHeaderXXSSProtection                  string `json:"_browser_header.xXSSProtection,omitempty"`
+
+	//todo add brute force detection
```

Comment:
Can you remove the todo comments and open an issue instead?  Or do you plan on implementing this within this PR?

Reply:
I plan on adding them later, but not within this PR, I will remove the comments